deployment/secure_research_environment/remote/secure_research_desktop/scripts/check_drive_mounts.sh

```diff
@@ -10,26 +10,27 @@ END="\033[0m"
 MOUNT_POINTS=("/data" "/home" "/scratch" "/shared" "/output")
 echo -e "${BLUE}Checking drives are mounted...${END}"
 for MOUNT_POINT in "${MOUNT_POINTS[@]}"; do
-    ls ${MOUNT_POINT}/* > /dev/null 2>&1
-    if [ "$(mount | grep $MOUNT_POINT)" ]; then
+    ls "${MOUNT_POINT}"/* > /dev/null 2>&1
+    if (mount | grep -q "$MOUNT_POINT"); then
```

Comment:
```suggestion
    if (findmnt $MOUNT_POINT); then
```

Reply:
Is `findmnt` a standard command? Good to know if so.

Reply:
It is part of `util-linux` so I would be surprised if it were not present.

Reply:
Done